README_PAYLOAD.md

```diff
@@ -61,21 +61,26 @@ todo
 ```
 Version is: 0001|0011 => 1.3 (signed message with chained signatures)  https://github.com/ubirch/ubirch-protocol/blob/master/README.md#field-types
 Type is: 0x54 (84)  https://github.com/ubirch/ubirch-protocol/blob/master/README.md#payload-type
-Example Payload:
-```
- [ "v1.0.2-PROD-20180326103205 (v5.6.6)",
-  6496,
-  3,
-  {
-    1594155266: 3665,
-    1594155326: 3682
-   },
-   {
-    "min": 3500,
-    "max": 4200,
-    "i": 60000,
-    "il": 1800000
-   }
+
+trackle message payload:
+```
+  [ // <msgpack.array[5]
+    "v1.0.2-PROD-20180326103205 (v5.6.6)", // <msgpack.raw> Version String
+    5, // <msgpack.uint32> wakeup cycles
+    3, // <msgpack.unit32> error state
```

Comment:
is there a documentation/link to the potential error states that exist?

Reply:
apparently there is not. Looking at the source code, the status consists of 4 bytes, where 6 different flags are OR-combined. 
0x03 means that the device shelf test has been executed and keys have been generated, i.e. 0x03 means the device is ready